lib/msal-common/src/utils/Constants.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * List of pre-established trusted host URLs. 
+ */
+export const AADTrustedHostList =  {
+    "login.windows.net": "login.windows.net",
+    "login.chinacloudapi.cn": "login.chinacloudapi.cn",
+    "login.cloudgovapi.us": "login.cloudgovapi.us",
+    "login.microsoftonline.com": "login.microsoftonline.com",
+    "login.microsoftonline.de": "login.microsoftonline.de",
+    "login.microsoftonline.us": "login.microsoftonline.us"
```

Comment:
I think this could be some other type (enum maybe?)

Reply:
changed string constant array